apps/src/lab2/levelEditors/aichatSettings/EditAichatSettings.tsx

```diff
@@ -214,10 +197,10 @@ const EditAiCustomizations: React.FunctionComponent<{
               </label>
               <Checkbox
                 name="hidePresentationPanel"
-                checked={aiCustomizations.hidePresentationPanel || false}
+                checked={aichatSettings.hidePresentationPanel || false}
```

Comment:
Nit: just curious - is there ever a case when this optional property is undefined since it's defined in `DEFAULT_LEVEL_AICHAT_SETTINGS`?

Reply:
Oh nope, there shouldn't be, I can remove this.

Reply:
Oh gotcha, yeah good question I suppose it doesn't have to be. Will update!